openreview/conference/templates/reviewerWebfield.js

```diff
@@ -183,10 +183,17 @@ var displayHeader = function(headerP) {
 
   var reducedLoadP = $.Deferred().resolve(0);
   if (REVIEW_LOAD > 0) {
+    userIds = _.union([user.profile.id], user.profile.emails);
```

Comment:
Should you also include all ~ ids in the names array? Or are those never used for these notes?

Reply:
good catch! solved.